src/models/launch_1/customization.md

```diff
@@ -0,0 +1,42 @@
+# Launch (Usage & Customization)
+
+The Launch Configurable Keyboard can be customized and personalized in a variety of ways. It is recommended to remove unplug all USB cables and devices from your Launch before changing keycaps or switches.
+
+- [Connecting and Using the Launch](#connecting-and-using-launch)
+- [Using the System76 Keyboard Configurator](#using-the-system76-keyboard-configurator)
+- [Remapping with QMK](#remapping-with-qmk)
+- [Removing and Installing Keycaps](#removing-and-installing-keycaps)
+- [Removing and Installing Switches](#removing-and-installing-switches)
+- [Removing the PCB](#removing-the-pcb)
+
+## Connecting and Using Launch
+
+![Launch ports](./img/launch-ports.png)
+
+Using either the USB-C to USB-C cable or the USB-A to USB-C cable, connect to your computer via your Launch's center USB type C port. Due to the directionality of USB, your Launch will only work when connected to your computer via the center port. Launch should now be ready to type on, and the USB ports on the back are ready to connect devices to.
+
+## Using the System76 Keyboard Configurator
```

Comment:
This is very much a software thing. Does this info belong here instead of in readme files within the configurator repository or a support article? (Not saying it doesn't, just bringing it up. We omitted the BIOS screenshots from tech docs for a reason.)

A potential reason it doesn't fit here is that the configurator, being software, will inevitably see updates, at which point this section would have to be updated; and if the configurator is used on future keyboard revisions, the software updates would still apply to older hardware revisions, so it would need to be updated in every section every time. Also, the configurator applies to Open Firmware laptops, so would we add this section to every Open Firmware laptop manual too?

Reply:
Makes sense. I've pulled out the configurator sections and will make a PR in support docs for those instead.